### PR TITLE
Exercises functions

### PR DESCRIPTION
**Pull request by D0LeD**

### CODE REVIEW

File: exercises/01_sort4/sort4.go [hunk 1]
```diff
@@ -24,4 +27,5 @@ func main() {
 	a, b, c, d = sort4(a, b, c, d)
 
 	fmt.Println("Sorted numbers:", a, b, c, d)
-}
+} //input: 2 6 3 7 | -3 1 0 -6 | 0 1 0 1 | 38 951 73 253 |
```
Comment (zayac) [inline exercises/01_sort4/sort4.go:30]: Now, when you know how to add tests for a program, you can modify `_test.go` directly and add those tests there.

Same applies for other programs.

File: exercises/04_power/power_test.go [hunk 2]
```diff
@@ -8,8 +8,20 @@ func TestPow(t *testing.T) {
 		x, y uint
 		want uint
 	}{
-		// Test cases.
+		{1, 1, 1},
```
Comment (zayac) [inline exercises/04_power/power_test.go:11]: A good idea is to add tests where `x` is negative.

**Comment by zayac** [comment]
Great commit descriptions!

File: exercises/04_power/power_test.go (hunk 2 above)
Comment (yarcat) [reply exercises/04_power/power_test.go:11]: And also both `x` and `y` are `0`

**Review by yarcat — CHANGES_REQUESTED** [review]
